api/app/src/routes/middlewares/auth.ts

```diff
@@ -1,8 +1,11 @@
 import base64url from "base64url";
 import { NextFunction, Request, Response } from "express";
-import { MAPIAccess } from "../../models/medical/mapi-access";
 import status from "http-status";
 
```

Comment:
nit: space between imports prevents prettier/organize imports to trigger, IIRC.

Reply:
I didnt know that, i usually like separating package imports vs local imports but if thats what we're going with then ill keep that approach going forward